ocean_only/otec1d/with_otec/MOM_input

```diff
@@ -164,44 +158,14 @@ Z_INIT_FILE_PTEMP_VAR = "PTEMP" ! default = "ptemp"
                                 ! The name of the potential temperature variable in TEMP_Z_INIT_FILE.
 Z_INIT_FILE_SALT_VAR = "SALT"   ! default = "salt"
                                 ! The name of the salinity variable in SALT_Z_INIT_FILE.
+Z_INIT_ALE_REMAPPING = True     !   [Boolean] default = False
+                                ! If True, then remap straight to model coordinate from file.
```

Comment:
It seems this input needs to be changed to correctly read in the input data.

I think that without this, the model assumes the input data is in layer coordinates and so it does funky things.